src/datalayer/store/promiseMiddleware.js

```diff
@@ -11,7 +13,7 @@ export default ({ dispatch, getState }) => (next) => (action) => {
 
   const beginAction = type;
   const successAction = `${type}_SUCCESS`;
-  const failureAction = `${type}_FAILURE`;
+  // const failureAction = `${type}_FAILURE`;
```

Comment:
restore

Reply:
@SpQuyt why resolve this when you not fixed it yet?

Reply:
Oh you did the "git reset --hard" or "git reset HEAD~"
I'm sure that I have resolved this one